### PR TITLE
Update set-up-multi-factor-authentication.md

### PR DESCRIPTION
**Pull request by msbemba**
Moved the TUrn off legacy per user MFA next to before you begin section.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/4858

File: microsoft-365/admin/security-and-compliance/set-up-multi-factor-authentication.md [hunk 1]
```diff
@@ -82,13 +90,7 @@ For most subscriptions modern authentication is automatically turned on, but if
 1. In the Microsoft 365 admin center, in the left nav choose **Settings** \> **Org settings**.
 2. Under the **Services** tab, choose **Modern authentication**, and in the **Modern authentication** pane, make sure **Enable Modern authentication** is selected. Choose **Save changes**.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/security-and-compliance/set-up-multi-factor-authentication.md:94]:
```suggestion
```

File: microsoft-365/admin/security-and-compliance/set-up-multi-factor-authentication.md [hunk 2]
```diff
@@ -82,13 +90,7 @@ For most subscriptions modern authentication is automatically turned on, but if
...
 
-### Turn off legacy per-user MFA
-
-If you have previously turned on per-user MFA, you must turn it off before enabling Security defaults.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/security-and-compliance/set-up-multi-factor-authentication.md:95]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Please remove blank lines.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@kwekuako Gentle reminder, please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@kwekuako conflict is resolved. Thanks

**Pull request merged**